WalletWasabi/WabiSabi/Client/ClientRound.cs

```diff
@@ -0,0 +1,35 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.Crypto;
+using WalletWasabi.WabiSabi.Backend.Rounds;
+using WalletWasabi.WabiSabi.Models.MultipartyTransaction;
+
+namespace WalletWasabi.WabiSabi.Client
+{
+	public record ClientRound
+	{
+		public ClientRound(Round round)
+		{
+			AmountCredentialIssuerParameters = round.AmountCredentialIssuerParameters;
+			VsizeCredentialIssuerParameters = round.VsizeCredentialIssuerParameters;
+			Id = round.Id;
+			FeeRate = round.FeeRate;
+		}
+
+		public uint256 Id { get; }
+		public FeeRate FeeRate { get; }
+		public CredentialIssuerParameters AmountCredentialIssuerParameters { get; }
+		public CredentialIssuerParameters VsizeCredentialIssuerParameters { get; }
+		public MultipartyTransactionState CoinjoinState { get; set; }
+		public TState Assert<TState>() where TState : MultipartyTransactionState =>
```

Comment:
@lontivero I had to duplicate this code because it was implemented in Backend.Round. Any idea how to get rid of the duplication?

Reply:
No yet. I can find a way by sure but not now. I mean, i don't like to have this duplication but I would prefer doing it in a different PR.